### ISSUE TITLE
React Native error with Object.assign polyfill

### ISSUE THREAD

**Issue by jaygarcia** [ProjectSeptemberInc/gl-react-native#2]
Hi, i'm trying to apply a shader to a GLView using the below code, but get the error (see after code). It seems that you're somehow not referencing styles appropriately, requiring the style value (see GL.View style=... below) be an object literal instead of a React JS reference able style instance.

Make sense?

```
    render() {
        var uniforms = {time: this.state.time};
        return (
            <GL.View style={styles.mainCt} uniforms={uniforms} width={100} height={200} shader={shaders.helloGL} >
                <GL.Target uniform="texture">
                    <View style={{flexDirection:'row', justifyContent : 'center'}}>
                        <Text style={styles.titleRed}>K</Text>
                        <Text style={styles.titleGreen}>ey</Text>
                        <Text style={styles.titleRed}>G</Text>
                        <Text style={styles.titleGreen}>en</Text>
                        <Text style={[styles.titleRed, {marginLeft: 10}]}>M</Text>
                        <Text style={styles.titleGreen}>usic</Text>
                    </View>
                    ...
                </GL.Target>
            </GL.View>
        );
    }
```

![image](https://cloud.githubusercontent.com/assets/651292/9584779/2ef3c2e4-4fe1-11e5-8e13-bcc94de316ac.png)

**Comment by gre** [comment]
Thanks for raising this.
fixed in `gl-react-native@0.1.5`

**Comment by jaygarcia** [comment]
Awesome dude! Thanks =D
